lib/matplotlib/style/core.py

```diff
@@ -71,18 +96,18 @@ def use(style):
 
     for style in styles:
         if not cbook.is_string_like(style):
-            mpl.rcParams.update(style)
+            _apply_style(style)
             continue
         elif style == 'default':
             mpl.rcdefaults()
```

Comment:
@mdboom, I think @weathergod has a good point here: 'default' style should modify only non-blacklisted things, shouldn't it?